docs/welcome/how-content-works.md

```diff
@@ -0,0 +1,187 @@
+# Understanding the Relationship Between Post Types, Taxonomies, and Custom Fields
+
+In WordPress, the true power of content modeling comes from combining **Post Types**, **Taxonomies**, and **Custom Fields**. Together, these tools allow you to build virtually any kind of data structure your website needs — from a real estate listing manager to a recipe library or an online product catalog.
+
+This guide will help you understand how each element works, when to use them, and how they interact with each other in practical projects — using movies as our central example.
+
+## Prerequisites
+
+-   SCF (Secure Custom Fields) installed and activated
+-   Administrator access to WordPress
+-   Basic understanding of WordPress concepts
+
+You can learn more about WordPress basics here:
+
+-   [Theme Basics](https://developer.wordpress.org/themes/basics/)
+-   [Plugin Basics](https://developer.wordpress.org/plugins/plugin-basics/)
+
+## 1. Post Types
+
+Post types define **what kind of content** you're managing. WordPress includes `Posts` and `Pages` by default, but you can create custom types like `Movies`, `Events`, or `Products`.
+
+**When to create a custom post type:**
+
+-   You want content that doesn’t belong under blog posts or pages
+-   You need a unique admin menu and editor for managing this content
+-   You want to separate things like `Movies` from standard posts
```

Comment:
How is this different than the first item? Maybe we should mention instead that we want to manage structured content.